docs/element-definitions.md

```diff
@@ -234,6 +239,49 @@ Note: At the moment only the HTTP protocol is supported.
 }
 ```
 
+### Affordance (Base API Element)
+
+An affordance element provides a way to provide affordance definitions for the
+API. These definitions can describe link relations used in a Transition Element,
+along with defining unsafe and idempotent transitions.
+
+While data structures are used for defining semantics around data models,
+affrodances are used for defining semantics around hypermedia state transitions.
+
+The control types used here are based on the types used in [ALPS][]. The meta
+property `id` SHOULD be used to define the machine-usable name of the
+affordance.
```

Comment:
Linking down to https://github.com/apiaryio/api-elements/pull/19/files#r96659051, I wonder if the `id` is not the right place to place the relation identifier. A `id` in API Elements is unique across the document, where as relation identifiers are generally just `unique` from where they are referenced. For example, there could me multiple occurrences of `next`, `prev`, `self` across multiple resources in an API Elements document. I don't believe using the `id` allows for this. Perhaps I misunderstand.